### DISCUSSION TITLE
dockerhub credentials docker-server?

### DISCUSSION THREAD

**Discussion by progS1m** [rancher-sandbox/rancher-desktop#1419] [Q&A] [answered]
I'm using rancher-desktop on mac os 12.2 (ARM).

Tried to add a dockerhub pull secret. But for some reason it did not work the usual way using **index.docker.io** as server.

I always got these errors in the log:
`Events:
  Type     Reason     Age                    From               Message
  ----     ------     ----                   ----               -------
  Normal   Scheduled  3m45s                  default-scheduler  Successfully assigned infra-common/demoapp-768ffc45f5-2j9pm to lima-rancher-desktop
  Normal   Pulling    2m13s (x4 over 3m45s)  kubelet            Pulling image "nginxinc/nginx-unprivileged:latest"
  Warning  Failed     2m12s (x4 over 3m44s)  kubelet            Failed to pull image "nginxinc/nginx-unprivileged:latest": rpc error: code = Unknown desc = Error response from daemon: Head "https://registry-1.docker.io/v2/nginxinc/nginx-unprivileged/manifests/latest": unauthorized: incorrect username or password`

Why is it trying to fetch from registry-1.docker.io?

Later I changed my secret to registry-1.docker.io and then all did work.

`kubectl -n infra-common create secret docker-registry dockerhub \
    --docker-server=registry-1.docker.io \
    --docker-username=<my username> \
    --docker-password=<my password>`

 Any ideas?

**Comment by jandubois** [comment, marked as answer]
idk, `registry-1.docker.io` is the official host name for Docker Hub, so this does not feel wrong to me.
